Automattic/crowdsignal-forms
# Change feedback block description

This PR updates the block description and adds an example to be rendered on the block library search preview

## Test instructions
Checkout and `make client`. Use the block library search to find the Feedback block, hovering it should show a preview with default values.

Known issue: the backdrop renders inside the preview somewhat quirky, but all this sizes and positioning are being skipped to be able to render the preview.
